extensions/license/main.ts

```diff
@@ -0,0 +1,14 @@
+import { LensMainExtension } from "@k8slens/extensions";
+import { shell } from "electron";
```

Comment:
This needs to double check. 
It's compiled to `require("electron")` in `main.js` but how it's eventually consumed inside Lens runtime with such call I'm not sure.
Currently after build with `yarn build:mac` and running `Lens.app` no one extension declared in `package.json ` not properly loaded..

Reply:
![image](https://user-images.githubusercontent.com/6377066/97577630-57c45300-19f8-11eb-9d4d-2d9501a9b70c.png)

Reply:
> This needs to double check.
> It's compiled to `require("electron")` in `main.js` but how it's eventually consumed inside Lens runtime with such call I'm not sure (works well in `development`-mode though).
> Currently after build with `yarn build:mac` and running `Lens.app` extensions declared in `package.json ` not loaded at all..

Could wrap and expose `shell. openExternal()` through the extensions api?

Reply:
> Seems like this is broken in `master` as well.
> 
> **Steps to reproduce**:
> 
> 1. `make build-extensions`
> 2. `yarn build:mac`
> 3. Run `dist/mac/Lens.app`
> 
seems to be fixed...

Reply:
Don't know what I'm doing wrong or is it some cached files.. but this is still happening on my machine (even after `master` merge)